go.sum

```diff
@@ -23,8 +23,7 @@ github.com/containerd/fifo v0.0.0-20191213151349-ff969a566b00 h1:lsjC5ENBl+Zgf38
 github.com/containerd/fifo v0.0.0-20191213151349-ff969a566b00/go.mod h1:jPQ2IAeZRCYxpS/Cm1495vGFww6ecHmMk1YJH2Q5ln0=
```

Comment:
Since you only updated Makefile, your change shouldn't need to modify go.sum/go.mod.

Reply:
Ok. I will revert this change.